src/ImageSharp/Formats/Jpg/JpegEncoderCore.cs

```diff
@@ -834,83 +833,107 @@ private void WriteStartOfScan<TColor>(PixelAccessor<TColor> pixels)
             this.Emit(0x7f, 7);
         }
 
+#pragma warning disable SA1201 // MethodShouldNotFollowAStruct
+
         /// <summary>
-        /// Encodes the image with no subsampling.
+        ///     Poor man's stackalloc for Encode420.
+        ///     This struct belongs to Encode420. Much easeier to understand code if they are close to each other. Why should I
```

Comment:
Good news: It might be refactored out of the encoder, because the decoder also has some cases, when it could be better to have a Block8x8F X 4 value type, than a `new  Block8x8F[4]` array.